crates/tauri-bundler/src/bundle/macos/ios.rs

```diff
@@ -178,7 +178,7 @@ fn generate_info_plist(
   writeln!(
     file,
     "  <key>CFBundleVersion</key>\n  <string>{}</string>",
-    settings.version_string()
+    settings.bundle_version()
```

Comment:
this iOS module here isn't used at all - we'll need a different logic to inject the bundle version for iOS

Reply:
ahh i see you already figured it out in iOS>build, nice